modules/apps/object/object-rest-test/src/testFunctional/macros/CustomObjectAPI.macro

```diff
@@ -28,9 +28,16 @@ definition {
 		}
 
 		if (isSet(scopeKey)) {
-			var scopeKey = JSONGroupAPI._getGroupIdByNameNoSelenium(
-				groupName = "Guest",
-				site = "true");
+			if (isSet(groupName)) {
```

Comment:
maybe we can update 
if(isSet(scopeKey)){
  if(!isSet(groupName)){
    var groupName = "Guest";
  }
  var scopeKey = ....
}

Reply:
will done